### PR TITLE
Feature flu pedestrian

### PR DESCRIPTION
**Pull request by joachimhgg**
- [x] (322) humans/pedestrian/protos/PedestrianRightHand.proto
- [x] (323) humans/pedestrian/protos/PedestrianLeftFoot.proto
- [x] (324) humans/pedestrian/protos/PedestrianLeftUpperLeg.proto
- [x] (325) humans/pedestrian/protos/PedestrianNeck.proto
- [x] (326) humans/pedestrian/protos/PedestrianLeftHand.proto
- [x] (327) humans/pedestrian/protos/PedestrianLeftUpperArm.proto
- [x] (328) humans/pedestrian/protos/PedestrianTorso.proto
- [x] (329) humans/pedestrian/protos/Pedestrian.proto
- [x] (330) humans/pedestrian/protos/PedestrianRightLowerArm.proto
- [x] (331) humans/pedestrian/protos/PedestrianRightUpperLeg.proto
- [x] (332) humans/pedestrian/protos/PedestrianRightFoot.proto
- [x] (333) humans/pedestrian/protos/PedestrianLeftLowerArm.proto
- [x] (334) humans/pedestrian/protos/PedestrianRightLowerLeg.proto
- [x] (335) humans/pedestrian/protos/PedestrianHead.proto
- [x] (336) humans/pedestrian/protos/PedestrianRightUpperArm.proto
- [x] (337) humans/pedestrian/protos/PedestrianLeftLowerLeg.proto
- [x] (014) projects/humans/pedestrian/worlds/pedestrian.wbt

### CODE REVIEW

**Comment by lukicdarkoo** [comment]
We recently discussed and a rule for converting robots is slightly different. All worlds in which the pedestrian (or any other robot we are converting) is used have to be converted to ENU. I forgot to mention it in the main PR.

That way, we demonstrate that the conversion is successful.

File: projects/humans/pedestrian/protos/Pedestrian.proto [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # template language: javascript
 
 PROTO Pedestrian [
-  field       SFVec3f    translation           0 1.27 0
+  field       SFVec3f    translation           0 0 1.27
```
Comment (lukicdarkoo) [inline projects/humans/pedestrian/protos/Pedestrian.proto:9]: This should be also changed in `docs/guide/humans.md`

File: projects/humans/pedestrian/worlds/pedestrian.wbt [hunk 2]
```diff
@@ -15,15 +14,16 @@ WorldInfo {
...
+  orientation 0.30689270390532825 -0.2902331601799681 -0.9064113751612043 4.648170330709373
+  position 0.005197140878764703 -22.232530148048017 16.570466173724824
   near 1
 }
 TexturedBackground {
 }
 TexturedBackgroundLight {
 }
 CircleArena {
+  rotation 0 0 1 1.5708003061004252
```
Comment (lukicdarkoo) [inline projects/humans/pedestrian/worlds/pedestrian.wbt:26]:
```suggestion
```
Do we need this?

**Review by lukicdarkoo — CHANGES_REQUESTED** [review]
Looks good, just a few minor comments

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**